.circleci/config.yml

```diff
@@ -0,0 +1,242 @@
+version: 2.1
+
+orbs:
+  android: circleci/android@0.2.1
+  react-native: react-native-community/react-native@4.4.2
+
+commands:
+
+  build-flutter-project:
+    description: "Install dependencies and build"
+    parameters:
+      working_directory:
+        default: .
```

Comment:
is this parameter automatically used?

Reply:
I cleaned it up a little bit since there were two commands doing the same. I don't think this parameter was being automatically used btw